packages/components/src/components.d.ts

```diff
@@ -84,7 +84,11 @@ export namespace Components {
   interface TInputText {}
   interface TTag {
     /**
-    * (optional) Button variant
+    * (optional) Tag on an <a> element
+    */
+    'link'?: boolean;
```

Comment:
link should accept a string f.e.: 'http://some-url.com' or '#some-id-to-jump-to'

Reply:
hi, i change to string, the code is outdated. can you refresh it?